bots/tests-scan

```diff
@@ -63,6 +63,10 @@ EXTERNAL_PROJECTS = {
     'cockpit-project/cockpit-podman': [
         'cockpit/fedora-28',
     ],
+    'weldr/welder-web': [
+        'cockpit/fedora-29/chrome',
+        'cockpit/fedora-29/firefox',
+    ],
```

Comment:
As discussed on IRC, can we please avoid the combinatorial explosion of "all browsers on all operating systems"? How about testing all browsers on F29, but only one browser on rhel-*?

Reply:
@martinpitt, sorry, forgot that parts. Updated.